tests/test_schema_rules.py

```diff
@@ -28,9 +28,15 @@ def test_lowercase_ids():
             ), "The id in {} should be lower case".format(schema["id"])
 
 
-def test_id_length():
+def test_nodeid_length():
+    prefix_len = len("node_")
+    postfix_len = len("_node_id_idx")
+    max_len = 63 - (prefix_len + postfix_len)
     for schema in dictionary.schema.values():
         if "id" in schema:
+            nodeid_len = sum(map(lambda x: len(x), schema["id"].split("_")))
```

Comment:
`len(schema["id"].replace("_", ""))` would be easier to read

Reply:
I don't know why I needed to make it that complicated